openstack/loadbalancer/v2/l7policies/requests.go

```diff
@@ -143,3 +143,47 @@ func Delete(c *gophercloud.ServiceClient, id string) (r DeleteResult) {
 	_, r.Err = c.Delete(resourceURL(c, id), nil)
 	return
 }
+
+// UpdateOptsBuilder allows extensions to add additional parameters to the
+// Update request.
+type UpdateOptsBuilder interface {
+	ToL7PolicyUpdateMap() (map[string]interface{}, error)
+}
+
+// UpdateOpts is the common options struct used in this package's Update
+// operation.
+type UpdateOpts struct {
+	// Name of the L7 policy.
+	Name string `json:"name,omitempty"`
```

Comment:
Is it possible to remove a name? If so, this should be `*string`. That will allow a value of `""` to be sent.

Reply:
That's a valid question, @jtopjian I tested in my octavia environment, an empty string for both name and description are allowed to be specified to update l7 policy, i will update the PR